### ISSUE TITLE
Incompatible Swift version 5.1.3 ?

### ISSUE THREAD

**Issue by goodhyun** [avast/topee#13]
Tried 2.2.0 for Swift 5.1.2, but it failed on 5.1.3

```
*** Downloading topee.framework binary at "swift 5.1.2"
***  Skipped installing topee.framework binary due to the error:
	"Incompatible Swift version - framework was built with 5.1.2 (swiftlang-1100.0.278 clang-1100.0.33.9) and the local version is 5.1.3 (swiftlang-1100.0.282.1 clang-1100.0.33.15)."

    Falling back to building from the source
*** Skipped building topee due to the error:
Dependency "topee" has no shared framework schemes
```

**Comment by Shinisaru** [comment]
Topee is not available for swift 5.1.3 as of now, please use 5.1.2

**Comment by goodhyun** [comment]
Thanks, but try on a 5.1.2 toolchain ended up with the similar error
```
"Incompatible Swift version - framework was built with 5.1.2 (swiftlang-1100.0.278 clang-1100.0.33.9) and the local version is 5.1.2 (swift-5.1.2-RELEASE)."
```
By the way, I thought there might be a way to solve this without reinstalling an old xcode. 
Carthage on the master seems to fix it : https://github.com/Carthage/Carthage/issues/2887
However, even it failed to proceed, and I suspect the following is related ? 
https://github.com/Carthage/Carthage/pull/2902#issuecomment-587230024

**Comment by pavelstudeny** [comment]
@goodhyun you are suggesting to use the **BUILD_LIBRARY_FOR_DISTRIBUTION** flag in Topee, correct?

**Comment by goodhyun** [comment]
Yes, if I understand it correctly, by going that way we can make Carthage not too picky on minor swift version change. 
https://www.donnywals.com/what-is-module-stability-in-swift-and-why-should-you-care/

**Comment by pavelstudeny** [comment]
Although this looks like a simple change, I won't get to it during the following two weeks, but I should have time to release a new version after that.

**Comment by pavelstudeny** [comment]
Should be fixed in 2.3.0. @goodhyun , I would be grateful if you could confirm that this fixes your problem.

**Comment by goodhyun** [comment]
Thanks @pavelstudeny 
Looks like it's working ! 😄

```
$ carthage update
Please update to the latest Carthage version: 0.34.0. You currently are on 0.33.0
*** Fetching topee
*** Checking out topee at "v2.3.0"
*** xcodebuild output can be found in /var/folders/mn/qs_0l7gn3qq6fpff7l7qw0280000gn/T/carthage-xcodebuild.1Ywzbl.log
*** Downloading topee.framework binary at "swift cross-minor version compatibility"

```

**Comment by eftalersoy** [comment]
now 5.2 update is problematic
*** Fetching topee
*** Checking out topee at "v2.3.0"
*** xcodebuild output can be found in /var/folders/lc/qm_88vyd4qq_s7mk14gqsg400000gn/T/carthage-xcodebuild.dhCreh.log
*** Downloading topee.framework binary at "swift cross-minor version compatibility"
***  Skipped installing topee.framework binary due to the error:
	"Incompatible Swift version - framework was built with 5.1.2 (swiftlang-1100.0.278 clang-1100.0.33.9) and the local version is 5.2 (swiftlang-1103.0.32.1 clang-1103.0.32.29)."

**Comment by sam-w** [comment]
@eftalersoy the problem should now be fixed with the release of Carthage 0.35.0

**Comment by pavelstudeny** [comment]
@sam-w perfect, thanks for a verification!
